### PR TITLE
Update TceForm.rst

### PR DESCRIPTION
**Pull request by michael-binder**
I think this is a typo it should be 'maxitems' instead of 'max'

### CODE REVIEW

File: Documentation/PageTsconfig/TceForm.rst [hunk 1]
```diff
@@ -293,13 +293,13 @@ config
    This property is available for various levels:
 
    table level, example:
-      `TCEFORM.tt_content.header.config.max`
+      `TCEFORM.tt_content.header.config.maxitems`
```
Comment (brotkrueml) [inline Documentation/PageTsconfig/TceForm.rst:296]: You have to distinguish between maxItems (= select items) and max characters (in input field). Have you looked into the code=

Comment (linawolf) [inline Documentation/PageTsconfig/TceForm.rst:296]:
```suggestion
      `TCEFORM.tt_content.header.config.max`
```

I agree with @brotkrueml , thi must be left unchanges. input has the property 'max' , see above

**Review by linawolf — CHANGES_REQUESTED** [review]
I agree with @brotkrueml

File: Documentation/PageTsconfig/TceForm.rst [hunk 2]
```diff
@@ -293,13 +293,13 @@ config
...
 
    table and record type level, example:
-      `TCEFORM.tt_content.header.types.textpic.config.max`
+      `TCEFORM.tt_content.header.types.textpic.config.maxitems`
 
    Flex form field level, example:
-      `TCEFORM.tt_content.pi_flexform.my_ext_pi1.sDEF.myField.config.max`
+      `TCEFORM.tt_content.pi_flexform.my_ext_pi1.sDEF.myField.config.maxitems`
```
Comment (linawolf) [inline Documentation/PageTsconfig/TceForm.rst:302]:
```suggestion
      `TCEFORM.tt_content.pi_flexform.my_ext_pi1.sDEF.myInputField.config.max`
```
I suggest we just make it more clear that this is an input field

**Review by linawolf — APPROVED** [review]

**Pull request merged**